components/common/carousel/carousel.tsx

```diff
@@ -12,7 +12,7 @@ type Props = {
     onNextSlideClick?: (slideIndex: number) => void;
     onPrevSlideClick?: (slideIndex: number) => void;
 }
-const Carousel: React.FC<Props> = ({
```

Comment:
this is just part of the campaign to remove the type React.FC from components that does not expect children